content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+Arguments:
+
+```
+--increment-instance-enabled=true
+--interval-time-between-checks=1000
+```
+
+Parameters:
+
+```
+app.basictransition.transition-sample.taskapp.exitMessage=FAILED
+```
+
+It should look like the following:
+
+![Transition Execution Flow Launch-Config](images/SCDF-composed-task-transition-launch-fail.png)
+Now that it has been executed let’s verify that path FAILED was actually followed and this can be done by pressing the `Executions` tab at the top of the task page:
+![Transition Execution Flow Launch-List](images/SCDF-composed-task-transition-launch-fail-list.png)
+
+This shows us that the `Composed Task Runner` controlling the composed task execution basic-transition was launched and the transition-sample was launched. From there the proper FAILED branch was executed as denoted by basictransition-timestamp-1 was launched.
+
+Now relaunch the `Composed Task Runner` and set the `taskapp.exitMessage` to `COMPLETED` to exercise the other branch. To do this let's select the `basictransition` to be executed as shown below:
+![Transition Execution Flow Launch](images/SCDF-composed-task-transition-launch.png)
+Now from the task launch page lets populate page with the following:
+
+Arguments:
+
+```
+--increment-instance-enabled=true
+--interval-time-between-checks=1000
+```
+
+Parameters:
+
+```
+app.basictransition.transition-sample.taskapp.exitMessage=COMPLETED
+```
+
+It should look like the following:
+
+![Transition Execution Flow Launch-Config-Complete](images/SCDF-composed-task-transition-launch-completed.png)
+
+Now that it has been executed let’s verify that path `COMPLETED` was actually followed and this can be done by pressing the `Executions` tab at the top of the task page:
+![Transition Execution Flow Launch-CompleteList](images/SCDF-composed-task-transition-launch-completed-list.png)
+
+#### What did I just do?
+
+Wait a minute…​ What is all that stuff I put in the command line? So for this example we wanted to show how to use both command line args and properties. We used the arguments to establish the properties for the `Composed Task Runner`:
+
+1. `--increment-instance-enabled=true` states that we want to be able to execute this Composed Task multiple times. (Composed Tasks are build using Spring Batch and thus are batch jobs)
+1. `--interval-time-between-checks=1000` states that the `Composed Task Runner` will wait 1 second between checks to make sure that a task is complete (the default is 10 seconds).
+
+We used the properties to set the transition-sample properties. Notice we had 4 sections for the taskapp.exitMessage property lets breakdown the `app.basictransition.transition-sample.taskapp.exitMessage=FAILED` into its components:
+
+1. `app` - tells spring cloud dataflow that this property is to be used for task application.
+1. `basictransition` - tells dataflow that this property will be used by the `Composed Task Runner` or one of the apps in the graph
```

Comment:
tells `D`ata `F`low to set a property on the application inside the `Composed Task Runner` application named `basictransition`